integrations/adobe-analytics/lib/index.js

```diff
@@ -46,7 +47,7 @@ var AdobeAnalytics = (module.exports = integration('Adobe Analytics'));
  * @param {*} value
  */
 
-AdobeAnalytics.sOption = function(field, value) {
+AdobeAnalytics.sOption = function (field, value) {
```

Comment:
Looks like there are a lot of diffs resulting from this styling change. Is it something auto-generated by your IDE? It would help to not have these diffs to reduce noise in the PR.